rpcs3/Emu/CPU/CPUTranslator.cpp

```diff
@@ -154,6 +154,16 @@ void cpu_translator::initialize(llvm::LLVMContext& context, llvm::ExecutionEngin
 		m_use_vnni = true;
 	}
 
+	// Test GFNI feature (TODO)
+	if (cpu == "tremont" ||
+		cpu == "gracemont" ||
+		cpu == "alderlake" ||
+		cpu == "raptorlake" ||
+		cpu == "meteorlake")
+	{
+		m_use_gfni = true;
```

Comment:
Why it only enumerates Intel processors?

Reply:
Only AMD processor which supports it is zen4, which supports AVX-512-Icelake, so it's set there instead.